### PR TITLE
Support displaying images data from `tui i`

### PR DESCRIPTION
**Pull request by eharris128**
What
===============
- Return `images` from `OnCommand` in the images command package
- Move the images model to the `images command` package
- Support navigation to the boilerplate debug model

Why
===============
- Enable entering the tui mode for images from either the `mint images --tui` or the `mint tui -> i` use modes
- Enables navigation among different models in `tui` mode

How Tested
===============
- `mint images` -> should not open tui mode
- `mint images --tui` -> should open tui mode
- `mint tui` -> opens tui mode -> `i` -> open image mode -> `esc` -> return to the home mode -> `d` -> open the debug view -> `q` -> quit the TUI

## Open Issues
- I turned down the logs that are output to the  CLI after exiting either the standalone TUI or the multi-model TUI mode. Both of these modes still output the `app` level logs to the CLI after the TUI closes.
- We want no logs to be output to the CLI after exiting TUI mode. This should  be addressed in a follow up MR:
image

## General questions
- Do we want to colocate the models associaqted with each command inside of the `command package`?
- Given the images `OnCommand ` outputs some `data` that the TUI model needs to ingest in order to perform an update, what is the preferred pattern to `await` the return of data from the call of `OnCommand`?
    - After fighting with race conditions, I took the path of modifying `OnCommand` in the images package to return `images.` My previous attempt had been to call `ImageModel.Update` (not exact reference), but because this call was returning before the `OnCommand` return, the final model returned inside of the previous`ImageModel.Update` call (being the initial call performed by the `Home Model 'i' key input handler` ) did not retain the images data. Having some intermediate state handler inside of the images model was another approach, but this felt more confusing to maintain.

### CODE REVIEW

**Comment by kcq** [comment]
`Given the images OnCommand  outputs some data that the TUI model needs to ingest in order to perform an update` <- ideally it'll be good to figure out how to use the execution context for the TUI as a way for the business logic in OnCommand to do I/O.

For example, for the `images` command `CLI.Action` (in `pkg/app/master/command/images/cli.go`) creates the command's execution context like this:
```
xc := app.NewExecutionContext(
			Name,
			gcvalues.QuietCLIMode,
			gcvalues.OutputFormat)
```
Then it passes `xc` to `OnCommand`.

It'll be good to figure out how to extend the `app.ExecutionContext` interface/struct to support what the TUI needs.

**Comment by kcq** [comment]
Waiting for results in `OnCommand` is doable (at least, in a straightforward way) in some of the commands (e.g., `images`), but other commands are interactive (e.g., `debug`) and waiting there until `OnCommand` returns doesn't quite work.

File: pkg/app/master/command/tui/cli.go [hunk 1]
```diff
@@ -18,7 +19,12 @@ var CLI = &cli.Command{
 	Aliases: []string{Alias},
 	Usage:   Usage,
 	Action: func(ctx *cli.Context) error {
-		m, _ := home.InitialModel()
+		gcvalues, ok := command.CLIContextGet(ctx.Context, command.GlobalParams).(*command.GenericParams)
+		if !ok || gcvalues == nil {
+			return command.ErrNoGlobalParams
+		}
+
+		m, _ := home.InitialModel(gcvalues)
```
Comment (kcq) [inline pkg/app/master/command/tui/cli.go:27]: let's use a less Bubble Tea specific name... just call it `InitialTUI` (`home.InitialTUI`).

File: pkg/app/master/command/images/model.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	"github.com/mintoolkit/mint/pkg/app/master/tui/common"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/keys"
+	"github.com/mintoolkit/mint/pkg/crt"
+	"github.com/mintoolkit/mint/pkg/crt/docker/dockerutil"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Model represents the state of the TUI.
+type Model struct {
```
Comment (kcq) [inline pkg/app/master/command/images/model.go:22]: Here as well... let's use a less Bubble Tea specific name... Let's call it `TUI` (`type TUI struct`). This way, somebody who doesn't know much about the Bubble Tea architecture and terminology will be able to understand what this code is for. Comments help too, of course. Calling the struct `TUI` will make it more obvious.

File: pkg/app/master/command/images/model.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+	OddRowStyle = CellStyle.Foreground(gray)
+	// EvenRowStyle is the lipgloss style used for even-numbered table rows.
+	EvenRowStyle = CellStyle.Foreground(lightGray)
+	// BorderStyle is the lipgloss style used for the table border.
+	BorderStyle = lipgloss.NewStyle().Foreground(white)
+)
+
+// End styles
+
+func LoadModel() *Model {
```
Comment (kcq) [inline pkg/app/master/command/images/model.go:52]: It'll be good to make it `LoadTUI`, so it's more obvious for casual observers

File: pkg/app/master/command/images/model.go [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+			return style
+		}).
+		Headers("Name", "ID", "Created", "Size").
+		Rows(rows...)
+
+	return *t
+}
+
+// InitialModel returns the initial state of the model.
+func InitialModel(images map[string]crt.BasicImageInfo, standalone bool) *Model {
```
Comment (kcq) [inline pkg/app/master/command/images/model.go:92]: Here as well... `InitialTUI` will make it easier to understand what it's for

File: pkg/app/master/command/tui/cli.go (hunk 1 above)
Comment (eharris128) [reply pkg/app/master/command/tui/cli.go]: Done. Resolved here - https://github.com/mintoolkit/mint/pull/55/commits/24ff10acd9fc80e73f91d19eed06a7e9f23e4c6e

File: pkg/app/master/command/images/model.go (hunk 3 above)
Comment (eharris128) [reply pkg/app/master/command/images/model.go]: Noted and addressed.

File: pkg/app/master/command/images/model.go (hunk 4 above)
Comment (eharris128) [reply pkg/app/master/command/images/model.go]: Noted and addressed.

File: pkg/app/master/command/images/model.go (hunk 2 above)
Comment (eharris128) [reply pkg/app/master/command/images/model.go]: Noted and addressed.

**Pull request merged**